packages/firestore/src/core/query.ts

```diff
@@ -533,18 +533,18 @@ export class RelationFilter extends Filter {
     }
   }
 
-  private matchesValue(value: FieldValue): boolean {
+  private matchesValue(docValue: FieldValue): boolean {
```

Comment:
How about "other"?  This actually matches what we have in [Android](https://github.com/firebase/firebase-android-sdk/blob/59af48cc23b5a23c29720d142f7aa2cd7036916a/firebase-firestore/src/main/java/com/google/firebase/firestore/core/RelationFilter.java#L74) and [iOS](https://github.com/firebase/firebase-ios-sdk/blob/4908f68a8feb0dc91bf6b01d93328508f8be0e4e/Firestore/core/src/firebase/firestore/core/relation_filter.h#L49) and will save you the effort of porting the rename. :-)